config/config.yaml

```diff
@@ -209,6 +209,12 @@ branch-protection:
           branches:
             master:
               protect: true
+        plugin-sdk-go:
+          required_pull_request_reviews:
+            required_approving_review_count: 2
```

Comment:
```suggestion
```

Since it's an incubating project, we could stick with only 1 approval at the moment.